<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WaiterDialog</class>
 <widget class="QDialog" name="WaiterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QCompanion - Waiter</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>5</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>7</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <item>
          <widget class="QLineEdit" name="EventTitle">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="placeholderText">
            <string>Event Title</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTimeEdit" name="EntryTime">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="displayFormat">
            <string>HH:mm:ss</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="CurrentTime">
           <property name="text">
            <string>Current Time</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="AddButton">
           <property name="text">
            <string>Add Event</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCalendarWidget" name="EntryDate">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QScrollArea" name="TimerScroller">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="TimerScrollerWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>612</width>
         <height>70</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>EventTitle</tabstop>
  <tabstop>EntryTime</tabstop>
  <tabstop>EntryDate</tabstop>
  <tabstop>AddButton</tabstop>
  <tabstop>TimerScroller</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>EventTitle</sender>
   <signal>returnPressed()</signal>
   <receiver>AddButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AddButton</sender>
   <signal>clicked()</signal>
   <receiver>WaiterDialog</receiver>
   <slot>onAddButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CurrentTime</sender>
   <signal>clicked()</signal>
   <receiver>WaiterDialog</receiver>
   <slot>onCurrentTimeClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
